docs/setup/independent/install-kubeadm.md

```diff
@@ -11,6 +11,7 @@ see the [Using kubeadm to Create a Cluster](/docs/setup/independent/create-clust
 {% endcapture %}
 
 {% capture prerequisites %}
+## Pre-Requisites
```

Comment:
The `{% capture prerequisites %}` seems to be a global rule, so if you're unhappy about how that's formatted, we should probably change it there

See https://github.com/kubernetes/website/blob/52a1b9f7ab84812faad4fd3ad7ad243cb6c3e65d/_includes/templates/task.md#L18

Reply:
revert modify, thanks